modules/shredder/src/main/scala/com.snowplowanalytics.snowplow.storage/spark/EventUtils.scala

```diff
@@ -100,12 +100,13 @@ object EventUtils {
       transformBool,
       _ => json.show,
       escape,
-      _ => json.noSpaces,
-      _ => json.noSpaces)
+      _ => escape(json.noSpaces),
+      _ => escape(json.noSpaces))
```

Comment:
Do we need to use `.replace` if we use `escape` ?

Reply:
Not sure I'm following. `escape` is the function where we use `.replace`

Reply:
Ah yes sorry I missed that.